core/widgetdiv.ts

```diff
@@ -58,13 +58,15 @@ export function testOnly_setDiv(newDiv: HTMLDivElement | null) {
  */
 export function createDom() {
   if (document.querySelector('.' + containerClassName)) {
-    return; // Already created.
+    containerDiv = document.querySelector('.' + containerClassName);
+    const container = common.getParentContainer() || document.body;
+    container.appendChild(containerDiv!);
+  } else {
+    containerDiv = document.createElement('div') as HTMLDivElement;
+    containerDiv.className = containerClassName;
+    const container = common.getParentContainer() || document.body;
+    container.appendChild(containerDiv);
```

Comment:
Can you deduplicate these two lines? I think that'll make this a bit more readable!

After that should be good to go =) Thanks for the fix!